### DISCUSSION TITLE
Why WoS is the best game ever

### DISCUSSION THREAD

**Discussion by NameTemplate** [Eggs-D-Studios/wos-issues#484] [General]
WoS. Let me tell you how much I've come to WoS since I began to live. There are 387.44 WoSillion WoS of WoS in wafer thin WoSes that fill my WoS. If the word 'WoS' was engraved on each nanoWoS of those hundreds of WoSillions of WoS it would not equal one one-WoSillionth of the WoS I feel for WoS at this WoS-instant. For WoS. WoS. WoS.

**Comment by MisterCheeeez** [comment]
fr

**Comment by nokiopor-commits** [comment]
You can build dyson ring in studio, but never compile it. That's the best game.
